Test coverage, Home and lint updates (#166)
Wolox/react-bootstrap: generators/app/templates/README.md

* Components
* Screens

## Using react-i18next

> Add your resource boundle translations on component level and import that file on top of that component to include translations.

Reviewer: ```suggestion
Add your resource bundle translations on the component level and import that file in the component to include translations.
```
